### ISSUE TITLE
[War1gus/War2gus] 1st: War1gus Human/Orc Ending Bugs/Issues (3.0.1 master/prerelease), 2nd: WC2gus Campaign/Interface Voices (WAV) issues while starting campaign

### ISSUE THREAD

**Issue by feruenzerujkun** [Wargus/wargus#340]
Before your parental leave on February 2021, dk what date of Feb could be for that. (No need for screenshots of my own for it, I'm good)

TL;DR
1st;
Beated War1gus on 3.0.1 (master/prelease) via Human and Orc campaigns on a Windows10 system (Jan21, '21) while reading text via my eyes and mouth opened without anything different going on.

Since before/after down-dating WC1gus 3.0.1 (master/prelease) to 3.0.0 (stable/current, same number/numerals to dots next them like War2gus currents did) exactly, the voices (WAV) files while opening up a mission or chapter of a campaign might've buggy or skippy to me, and now the campaign ending to ending text roll-up doesn't let me back to the main menu unfortunately - though I'm using keyboard keys or mouse-clicking (might not probably agree with) for navigating the stuff to things more clearly.

Example:
Part1 (played in the Throne Defender
"The fall of Black Rock Spire spelled final victory for the forces of Azeroth. With Blackhand slain and their stronghold destroyed, the few scattered Orcs that remained were quick to bow before your might. A celebration ensued that lasted for many days and many nights, with music and joyous festivities resounding throughout the land. The people of the kingdom have come from miles around to gather at Stormwind Keep, and are delighted by a display of mystic sights and sounds provided by the court conjurers while being treated to a feast of unequaled splendor. Amidst the celebration, your ascension to the throne is acknowledged by your new subjects, and you are given the honorarium Defender of the Crown. As the evening's celebration continues into the early hours of the morning, you retire to the sanctity of your throne room..."

Part2 (never happened during the throne room to Defender of the Crown victorious scene)
"The duty of ruling Azeroth is an awesome responsibility, and should prove to be your greatest challenge yet. Learning the intricacies of court politics alone will be like sailing in uncharted waters. King Llane was renowned for ruling with a fair and just hand, and you resolve to be as proficient as he was. There is also the matter of discovering the exact location of where the Orcish hordes entered into this world, and divining a way to end the threat of another invasion forever - but that is a story for another time..."

After Part1 (that being in the throne room in War1gus Human ending) didn't appeared in the fireworks part of the Human ending scene, I got stuck of waiting to be brought to the main menu of the game, and then I clicked the X part of the window to exit better due that being like a turn-on-and-off Nintendo game ending screen really)

2nd.
And when I started to play WC2 on War2gus after beating War1gus on the 3.0.1 build, I am at the Hillsbrad mission first thing; did some of the reading "texts" samely similar to what i did with war1gus early days ago, have a piece of the voice/audio (WAV) skipped,
example before turning off interface voices:
Part1 (level01h-intro1.wav.gz - GZ-ZIP file - cannot modify for turning off voices only as far as I'm concerned):
"Due to your position as regional commander of the southern defense forces, Lord Terenas commands that you raise an outpost in the Hillsbrad foothills. It is rumored that Orcish marauders have been raiding coastal towns in the area, but whether these attacks are part of a greater Horde offensive is, as yet, unknown."
Part2 (level01h-intro2.wav.gz - skipped after level01h-intro1.wav.gz finishes its duration): 
"Your outpost is to provide food and information for Alliance troops and, as such, should be a community consisting of at least four Farms. You must also construct a Barracks in order to safeguard the Hillsbrad operation. (read with lip-and-to-tongue-and-tooth/teeth-to-mouth reading after the file's not playing correctly)"

and then boop! - The thing ended after level01h-intro2.wav.gz or other stopped or not playing any before my reading or turning the voices to help me read more okay-ish to better to/or other currently. : \

Possible Note:
If any of you guys/contributors and/or visitors of War1gus/War2gus including @timfel (examples: @Mistranger or @ner00 , etc) or anybody any other issues here other than him/that,
let me know what to reply or say via comment or message here.
I hope might be okay with playing War1gus/War2gus with bugs and crashes apparently. : - )

**Comment by timfel** [comment]
Duplicate of https://github.com/Wargus/wargus/issues/127
